docs/api/asyncio_con.rst

```diff
@@ -539,6 +535,51 @@ Connection Pools
         See :py:meth:`AsyncIOConnection.execute()
         <edgedb.AsyncIOConnection.execute>` for details.
 
+    .. py:method:: retry()
+
+        Create a :py:class:`AsyncIORetry` object. This is a way to make
```

Comment:
Can we please copy the text from https://github.com/edgedb/edgedb-python/pull/150/files#diff-833af12d3f57fe855ba1925cc6b4409cef2094fb7aac456676721f0560e1e435R237?